addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -1643,7 +1644,7 @@ export function isSelectionFormat(editable, format) {
     const selectedNodes = getTraversedNodes(editable)
         .filter(n => n.nodeType === Node.TEXT_NODE);
     const isFormatted = formatsSpecs[format].isFormatted;
-    return selectedNodes.length && selectedNodes.every(n => isFormatted(n, editable));
+    return selectedNodes.length && selectedNodes.every(n => n.nodeValue === INVISIBLE_BOM_CHAR || isFormatted(n, editable));
```

Comment:
I'm confused as to why `\ufeff` is considered differently than `\u200b` for the purpose of this check since both are invisible. The only difference between the two is whether the browser can break a line at their position or not. I would have expected this check to be on `ZERO_WIDTH_CHARS` instead, which would have the added benefit of not introducing a new constant. (By the way, what do you mean by "BOM" in this context?)

Reply:
Hello, I updated the PR, but I'm still only checking for `\ufeff`. The reason is that after testing different scenarios, I noticed that `\u200b` is used when applying a style to an empty selection and then starting to type (for example, pressing **Ctrl+B**). So, that character should be checked for styling, while `\ufeff` is never styled (at least from what I observed). Since they are used differently, they should be treated separately.

wdyt? By the way, I took the name from Wikipedia because I wasn't sure what else to call it.